Lib/Emscripten/Emscripten.cpp

```diff
@@ -742,6 +747,20 @@ void Emscripten::initializeGlobals(Context* context,
 			if(function) { Runtime::invokeFunctionChecked(context, function, {}); }
 		}
 	}
+
+	// Store ___errno_location.
+	Function* errNoLocation
+		= asFunctionNullable(getInstanceExport(moduleInstance, "___errno_location"));
+	if(errNoLocation
+	   && getFunctionType(errNoLocation)
+			  == FunctionType(TypeTuple{ValueType::i32}, TypeTuple{}))
+	{
+		IR::ValueTuple errNoResult = Runtime::invokeFunctionChecked(context, errNoLocation, {});
+		if(errNoResult.size() == 1 && errNoResult[0].type == ValueType::i32)
+		{
+			emscriptenErrNoLocation = errNoResult[0].i32;
```

Comment:
Not sure if this should be casted to `(U32)`?

Reply:
Not necessary. What you need to be careful to avoid is implicitly cast `I32` to an `Iptr` or `I64`. That would sign extend it instead of zero extend it.